core/tenant/handler/v1beta1/project.go

```diff
@@ -28,7 +28,7 @@ type ProjectService interface {
 type TenantService interface {
 	GetDetails(ctx context.Context, tnnt tenant.Tenant) (*tenant.WithDetails, error)
 	GetSecrets(ctx context.Context, tnnt tenant.Tenant) ([]*tenant.PlainTextSecret, error)
-	GetSecret(ctx context.Context, tnnt tenant.Tenant, name string) (*tenant.PlainTextSecret, error)
+	GetSecret(ctx context.Context, tnnt tenant.Tenant, name tenant.SecretName) (*tenant.PlainTextSecret, error)
```

Comment:
Why we need this change ?

Reply:
to ensure that SecretName is always in the right format i.e. Capital case